src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml

```diff
@@ -38,16 +38,9 @@
             <argument type="collection" />
         </service>
 
-        <!--
-            If you want to change the Request class, modify the code in
-            your front controller (app.php) so that it passes an instance of
-            YourRequestClass to the Kernel.
-            This service definition only defines the scope of the request.
-            It is used to check references scope.
-
-            This service is deprecated, you should use the request_stack service instead.
-        -->
-        <service id="request" scope="request" synthetic="true" synchronized="true" />
+        <service id="request" scope="request" synthetic="true" synchronized="true">
```

Comment:
are you sure ? I found one here https://github.com/symfony/symfony/blob/c06db308b0c889ec6266b66f4220aa8578a65d3c/src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php#L318
but the message is different and I thought we should keep both. I did not find anything else, did I miss something?